README.md

```diff
@@ -1,65 +1,7 @@
-# RAT
-Reflectivity Algorithms Toolbox for Rascal
+[![license](https://img.shields.io/github/license/Reflectivity-Algorithms-Toolbox/RAT.svg)](https://github.com/Reflectivity-Algorithms-Toolbox/RAT/blob/master/LICENSE)
+[![release](https://img.shields.io/github/release/Reflectivity-Algorithms-Toolbox/RAT.svg)](https://github.com/Reflectivity-Algorithms-Toolbox/RAT/releases)
+RAT
+===
+RAT is an acronym for Reflectivity Algorithms Toolbox for Rascal This is a MATLAB toolbox for analysing non-polarized neutron reflectivity data at multiple contrasts. This is designed to fit multiple contrast neutron reflectivity data, primarily using the Abeles layer model but user-defined model function are also supported.
```

Comment:
I think a full stop needs to be added after "Rascal" in "RAT is an acronym for Reflectivity Algorithms Toolbox for Rascal This is a MATLAB toolbox . . ."